core/services/standardcapabilities/standard_capabilities.go

```diff
@@ -74,27 +79,36 @@ func (s *standardCapabilities) Start(ctx context.Context) error {
 			return fmt.Errorf("error starting standard capabilities service: %v", err)
 		}
 
-		if err = s.capabilitiesLoop.WaitCtx(ctx); err != nil {
-			return fmt.Errorf("error waiting for standard capabilities service to start: %v", err)
-		}
+		s.wg.Add(1)
+		go func() {
+			defer s.wg.Done()
+			cctx, cancel := s.stopChan.NewCtx()
+			defer cancel()
 
-		if err = s.capabilitiesLoop.Service.Initialise(ctx, s.spec.Config, s.telemetryService, s.store, s.CapabilitiesRegistry, s.errorLog,
-			s.pipelineRunner, s.relayerSet, s.oracleFactory); err != nil {
-			return fmt.Errorf("error initialising standard capabilities service: %v", err)
-		}
+			if err = s.capabilitiesLoop.WaitCtx(cctx); err != nil {
```

Comment:
So what is going to happen now if the LOOPP can't be started? Are we going to hang here forever?

Reply:
(where forever == 3 minutes)